components/eam/src/physics/cam/clubb_intr.F90

```diff
@@ -3181,6 +3268,41 @@ subroutine clubb_surface (state, cam_in, ustar, obklen)
                         kinheat, kinwat, kbfs, obklen(i) )
     enddo
 
+    if (use_od_ss) then
+    !add calculation of bulk richardson number here
+    !
+    !compute the whole level th and thv for diagnose of bulk richardson number
+    thv_lv=0.0_r8
+    th_lv=0.0_r8
+
+    !use the same virtual potential temperature formula as above (thv) except for all vertical levels
+    !used for bulk richardson number below in pblintd_ri
+    do i=1,ncol
+      do k=1,pver
+         th_lv(i,k) = state%t(i,k)*state%exner(i,k)
+         if (use_sgv) then
+           thv_lv(i,k) = th_lv(i,k)*(1.0_r8+zvir*state%q(i,k,ixq) &
+                    - state%q(i,k,ixcldliq))  
+         else
+           thv_lv(i,k) = th_lv(i,k)*(1.0_r8+zvir*state%q(i,k,ixq))
+         end if
+      enddo
+    enddo
+
+    !recalculate the kbfs stored in kbfs_pcol for bulk richardson number in pblintd_ri
+    kbfs_pcol=0.0_r8
+    do i=1,ncol
+        call calc_ustar( state%t(i,pver), state%pmid(i,pver), cam_in%wsx(i), cam_in%wsy(i), rrho, ustar(i) )
+        call calc_obklen( th(i), thv(i), cam_in%cflx(i,1), cam_in%shf(i), rrho, ustar(i), &
+                        kinheat, kinwat, kbfs, obklen(i) )
+        kbfs_pcol(i)=kbfs
+    enddo
+
+    !calculate the bulk richardson number
+    call pblintd_ri(ncol, gravit, thv_lv, state%zm, state%u, state%v, &
+                ustar, obklen, kbfs_pcol, state%ribulk)
+    endif
```

Comment:
Here's another place with inconsistent indentation - I know we don't have an "official" policy on code indentation, but I think it's common sense that the code within an if block should be indented